content/en/agent/faq/how-do-i-uninstall-the-agent.md

```diff
@@ -176,7 +176,7 @@ Uninstall the Agent using Add/Remove Programs.
 Alternatively, use the Powershell command below. **Note**: Some Agent versions may cause a forced reboot. To prevent this, add a no restart parameter like `REBOOT=ReallySuppress`.
```

Comment:
I think we can remove the line asking users to ` add a no restart parameter like `REBOOT=ReallySuppress`.` since we did that for them already.